distribution/lib/Standard/Base/0.0.0-dev/src/Main.enso

```diff
@@ -42,6 +44,8 @@ export project.Errors
 export project.IO
 export project.Math
 export project.Meta
+export project.Meta.Enso_Project
```

Comment:
Is it important to export `Enso_Project` symbol? Isn't it enough to have just `enso_project`?

Reply:
I am afraid so, this is the only way how to export a static method from a module. See for example, https://github.com/enso-org/enso/tree/develop/engine/runtime/src/test/resources/Test_Multiple_Exports.

Reply:
Hmm, I have just tried removing the import/export of the module, and everything seems to work, even on the IDE side. Commit: 1843fdc5dda5ba9c8b8e5c49321f330f73ec1aa3